app/assets/javascripts/workarea/storefront/paypal/modules/paypal_buttons.js

```diff
@@ -49,23 +49,15 @@ WORKAREA.registerModule('paypalButtons', (function () {
             paypal
             .Buttons(getConfig())
             .render($container[0]);
-        },
+        };
 
-        /**
-         * @method
-         * @name init
-         * @memberof WORKAREA.paypalButtons
-         */
-        init = function ($scope) {
+        WORKAREA.paypal.ready(function ($scope) {
             var $buttonContainer = $('#paypal-button-container', $scope);
 
-            if (window.paypal === undefined) { return; }
             if (_.isEmpty($buttonContainer)) { return; }
 
             setup($buttonContainer);
-        };
+        });
 
-    return {
-        init: init
-    };
+    return {};
```

Comment:
do I need this at the bottom, or can these modules just not return anything and that's OK?

Reply:
I dunno, I would probably do this as a just-in-case:
```
  return {
    init: $.noop
  }
```

Reply:
Just confirmed that removing this doesn't seem to have an effect, so I'm doing it.